o2.sh

```diff
@@ -28,6 +27,7 @@ requires:
   - KFParticle
   - VecGeom
 build_requires:
+  - FairMQ
```

Comment:
mmm fairmq is definitely needed at runtime. Am I missing something?

Reply:
If you look at modulefile it is not loaded, it probably comes with FairRoot so just to make it consistent I wanted to add an entry to modulefile OR as above, move `FairMQ` to `build_requires` (it will be compiled anyway as it comes with FairRoot).

Reply:
I think we should move it to the modulefile.